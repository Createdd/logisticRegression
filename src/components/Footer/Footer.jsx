import React from "react";

const Footer = () => (
  <footer className="page-footer grey darken-4">
    <div className="container blue-grey-text">
      <div className="col s6">
        <h6>Visualize Regression</h6>
        <h5>
          written by <strong>Daniel Deutsch</strong>
        </h5>
      </div>
    </div>
    <div className="footer-copyright">
      <div className="container">
        <a
          href="http://createdd.com"
          target="_blank"
          className="blue-grey-text text-lighten-3"
          rel="noreferrer noopener"
        >
          Homepage /
        </a>
        <a
          href="https://github.com/DDCreationStudios"
          target="_blank"
          className="blue-grey-text text-lighten-3"
          rel="noreferrer noopener"
        >
          Github /
        </a>
        <a
          href="https://www.linkedin.com/in/daniel-deutsch-b95611127"
          target="_blank"
          className="blue-grey-text text-lighten-3"
          rel="noreferrer noopener"
        >
          LinkedIn /
        </a>
        <a
          href="https://medium.com/@ddcreationstudi"
          target="_blank"
          className="blue-grey-text text-lighten-3"
          rel="noreferrer noopener"
        >
          Medium /
        </a>
        <a
          href="https://hashnode.com/@DDCreationStudio"
          target="_blank"
          className="blue-grey-text text-lighten-3"
          rel="noreferrer noopener"
        >
          Hashnode /
        </a>
        <a
          href="https://www.freecodecamp.com/ddcreationstudios"
          target="_blank"
          className="blue-grey-text text-lighten-3"
          rel="noreferrer noopener"
        >
          FreeCodeCamp /
        </a>
        <a
          href="https://codepen.io/ddcreationstudios/"
          target="_blank"
          className="blue-grey-text text-lighten-3"
          rel="noreferrer noopener"
        >
          CodePen
        </a>
      </div>
    </div>
  </footer>
);

export default Footer;
