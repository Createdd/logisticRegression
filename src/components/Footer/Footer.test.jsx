import * as React from "react";
import * as renderer from "react-test-renderer";

import Footer from "./Footer";

describe("Test Footer", () => {
  it("renders snapshot", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
