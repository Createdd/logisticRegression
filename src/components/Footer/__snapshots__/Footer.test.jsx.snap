// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer renders snapshot 1`] = `
<footer
  className="page-footer grey darken-4"
>
  <div
    className="container blue-grey-text"
  >
    <div
      className="col s6"
    >
      <h6>
        Visualize Regression
      </h6>
      <h5>
        written by 
        <strong>
          Daniel Deutsch
        </strong>
      </h5>
    </div>
  </div>
  <div
    className="footer-copyright"
  >
    <div
      className="container"
    >
      <a
        className="blue-grey-text text-lighten-3"
        href="http://createdd.com"
        rel="noreferrer noopener"
        target="_blank"
      >
        Homepage /
      </a>
      <a
        className="blue-grey-text text-lighten-3"
        href="https://github.com/DDCreationStudios"
        rel="noreferrer noopener"
        target="_blank"
      >
        Github /
      </a>
      <a
        className="blue-grey-text text-lighten-3"
        href="https://www.linkedin.com/in/daniel-deutsch-b95611127"
        rel="noreferrer noopener"
        target="_blank"
      >
        LinkedIn /
      </a>
      <a
        className="blue-grey-text text-lighten-3"
        href="https://medium.com/@ddcreationstudi"
        rel="noreferrer noopener"
        target="_blank"
      >
        Medium /
      </a>
      <a
        className="blue-grey-text text-lighten-3"
        href="https://hashnode.com/@DDCreationStudio"
        rel="noreferrer noopener"
        target="_blank"
      >
        Hashnode /
      </a>
      <a
        className="blue-grey-text text-lighten-3"
        href="https://www.freecodecamp.com/ddcreationstudios"
        rel="noreferrer noopener"
        target="_blank"
      >
        FreeCodeCamp /
      </a>
      <a
        className="blue-grey-text text-lighten-3"
        href="https://codepen.io/ddcreationstudios/"
        rel="noreferrer noopener"
        target="_blank"
      >
        CodePen
      </a>
    </div>
  </div>
</footer>
`;
